### DISCUSSION TITLE
Conversion of SP Chemnitz demo set

### DISCUSSION THREAD

**Discussion by wsc44** [GrandOrgue/OdfEdit#19] [General]
Conversion of Chemnitz demo set with V 2.3 gave mixed results. Console and mixer panels are O.K., single jamb panel shows Forte/Tutti/Generalkoppel elements with partial functionality and L/R jamb, pedal matrix and crescendo panels do not display any elements. - There seems to be a problem in feeding the correct data to the loop which generates the element and swirches entries e.g. panel001 (single jamb) consists of 41 elements, but these elements are a repetition of Tutti/Generalkoppel/Forte entries. And all these Tutti/Generalkoppel/Forte elements use the same switch001.

**Comment by eturpault** [comment]
Interresting sample set, thank you, with a complex structure of conditional switches, especially due to the presence of the crescendo and other evoluated controls. OdfEdit doesn't support yet such switches, it is the reason why there is this specially bad ODF conversion result in term of stops control. The sharp keys of the keyboards are not displayed correctly, a fix is in progress. So for the moment this organ is not playable in GO.
I am working on implementing the support of the conditional switches, this sample set will be a good test case. Please be patient, it should take me a couple of months before to release it, depending on my available time, it is not an easy matter.
Anyway it think that it will not be possible to convert in GO all the controlling features that HW has. I will see what I can do.

**Reply by eturpault** [reply]
In the generated GO ODF, you can set DefaultToEngaged=Y in the section [Switch001], it permits to play a tutti of the manual I.
